datasync/chaindatafetcher/kafka/repository.go

```diff
@@ -26,11 +26,24 @@ import (
 	"github.com/klaytn/klaytn/datasync/chaindatafetcher/types"
 )
 
-type blockTraceResult struct {
+type traceGroupResult struct {
 	BlockNumber      *big.Int              `json:"blockNumber"`
 	InternalTxTraces []*vm.InternalTxTrace `json:"result"`
```

Comment:
`json:"result"` for `InternalTxTraces`?

Reply:
Yes, it is the result of a block tracing.